graph/typeDefs.graphql

```diff
@@ -339,6 +339,9 @@ type Comment {
 
   # describes how the comment can be edited
   editing: EditInfo
+
+  # Indicates if it has a parent
+  hasParent: Boolean
```

Comment:
We want to encourage people to batch requests together. It's my opinion that instead, we do not provide the parent id at the top level.

It would encourage the use of the public api in a desired way, and force the user to think about batching because they can't get the ID otherwise.